packages/ses/test/test-compartment-make-scope-proxy.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-underscore-dangle */
+import test from 'ava';
+import '../index.js';
+
+lockdown();
+
+test('compartment main use case exploded', t => {
+  function power(a) {
+    return a + 1;
+  }
+
+  const c1 = new Compartment({ power });
+  const { scopeProxy: scopeProxy1 } = c1.__makeScopeProxy__();
+  const power1 = scopeProxy1.power;
+  const attenuatedPower = arg => {
+    if (arg <= 0) {
+      throw new TypeError('only positive numbers');
+    }
+    return power1(arg);
+  };
+
+  const c2 = new Compartment({ power: attenuatedPower });
+  const { scopeProxy: scopeProxy2 } = c2.__makeScopeProxy__();
+  const power2 = scopeProxy2.power;
+  const use = arg => {
+    return power2(arg);
+  };
```

Comment:
We probably need to do something to validate the “allow one use of eval” on the scope controller. We need to validate this API end-to-end with the motivating feature, so perhaps using `StaticModuleRecord` to build a functor source string, wrap it with the magic lines, inject a scope proxy, verify its containment.

Reply:
my intention was to not allow any eval for this scopeProxy at this time

Reply:
is the correct way to enable then disable it? or never enable it?